justfile

```diff
@@ -72,50 +72,10 @@ slt-bin-debug *args:
 
 # Run SQL Logic Tests over RPC
 rpc-tests:
-  just slt --protocol=rpc \
-    'sqllogictests/cast/*' \
-    'sqllogictests/cte/*' \
-    'sqllogictests/functions/delta_scan' \
-    'sqllogictests/functions/generate_series' \
-    'sqllogictests/functions/version' \
-    'sqllogictests/joins/*' \
-    'sqllogictests/topn/*' \
-    'sqllogictests/window/*' \
-    'sqllogictests/aggregates' \
-    'sqllogictests/alter' \
-    'sqllogictests/create_table' \
-    'sqllogictests/credentials' \
-    'sqllogictests/csv' \
-    'sqllogictests/debug' \
-    'sqllogictests/delete' \
-    'sqllogictests/demo_pg' \
-    'sqllogictests/drop' \
-    'sqllogictests/explain' \
-    'sqllogictests/external_table' \
-    'sqllogictests/http' \
-    'sqllogictests/infer' \
-    'sqllogictests/information_schema' \
-    'sqllogictests/metabase' \
-    'sqllogictests/name' \
-    'sqllogictests/object_names' \
-    'sqllogictests/pg_catalog' \
-    'sqllogictests/rpc' \
-    'sqllogictests/schema' \
-    'sqllogictests/search_path' \
-    'sqllogictests/select' \
-    'sqllogictests/simple' \
-    'sqllogictests/table' \
-    'sqllogictests/temp_table' \
-    'sqllogictests/time' \
-    'sqllogictests/tunnels' \
-    'sqllogictests/update' \
-    'sqllogictests/vars' \
-    'sqllogictests/views' \
-    'sqllogictests/virtual_catalog' \
-    'sqllogictests/xlsx' \
-    'sqllogictests/prql' \
-    'sqllogictests/describe_rpc' \
-    'sqllogictests/allowed_operations'
+  just slt --protocol=rpc "sqllogictests/*" \
+    --exclude "sqllogictests/functions/cache_external_database_tables" \
+    --exclude "sqllogictests/functions/kdl" \
```

Comment:
I have no idea why this wouldn't work. Seems suspect.

Reply:
We're just passing in the default function registry here: https://github.com/glaredb/glaredb/blob/f0da39cb5c2d2453b7aee3e5ea27f14af785b108/crates/rpcsrv/src/session.rs#L58-L62

It's a trait that we can implement, but the `TaskContext` also implements it (which you get from a datafusion session context).